.dockerignore

```diff
@@ -1,4 +1,31 @@
+# exclude all files and directories
+*
+
+# include only the globs we want
+!**/*.dockerignore
```

Comment:
Are these essentially a double negative, during `.dockerignore` into a sort of ` .dockerinclude`?

Reply:
indeed